ICAFIX/scripts/computeRecleanFeatures.m

```diff
@@ -0,0 +1,285 @@
+        end
+        probability=prob_file.data(1:size(sICA,2),:);
+        Stats=load([SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_hp' hp '.ica/filtered_func_data.ica/melodic_ICstats']);
+        FeaturesFile=load([SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_hp' hp '.ica/fix/features.csv']);
+        Powerspectra=load([SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_hp' hp '.ica/filtered_func_data.ica/melodic_FTmix']);
+
+        % derive TR
+        file_name=[SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_Atlas.dtseries.nii'];
+        CIFTIDenseTimeSeries=ciftiopen(file_name, "wb_command");
+        TR = CIFTIDenseTimeSeries.diminfo{2}.seriesStep;
+
+        PowerspectraScale=[(1/(TR*2))/size(Powerspectra,1):(1/(TR*2))/size(Powerspectra,1):1/(TR*2)];
+        StandardPowerspectraScale=[(1/(TR*2))/(RunsXNumTimePoints/2):(1/(TR*2))/(RunsXNumTimePoints/2):1/(TR*2)];
+        sICAweighted=sICA.*repmat(Stats(:,1)',length(sICA),1);
+        sICAnoisy=sum(abs(sICA),2);
+        sICAnoisy=sICAnoisy>prctile(sICAnoisy,87.5);
+
+        CIFTI=ciftiopen([SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_hp' hp '.ica/filtered_func_data.ica/melodic_oIC.dscalar.nii'],'wb_command');
+        CIFTIVN=ciftiopen([SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_Atlas_hp' hp '_clean_vn.dscalar.nii'],'wb_command');
+        CIFTI.cdata=CIFTI.cdata./repmat(CIFTIVN.cdata,1,size(CIFTI.cdata,2));
+        %unix(['/media/myelin/brainmappers/Connectome_Project/YA_HCP_Final/Scripts/MSMAllResampleAndSmoothSICA.sh ' SubjFolderlist ' ' subj ' ' fMRIName ' ' hp ' ' Resolution ' ' num2str(FinalSpatialSmoothingFWHM) ' wb_command']);
+        CIFTI_file=[SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_hp' hp '.ica/filtered_func_data.ica/melodic_oIC.dscalar.nii'];
+        CIFTIMSMAll_file=[SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_hp' hp '.ica/filtered_func_data.ica/melodic_oIC_MSMAll.dscalar.nii'];
+        CIFTIMSMAll_smooth_file=[SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_hp' hp '.ica/filtered_func_data.ica/melodic_oIC_MSMAll_4.dscalar.nii'];
+        fs_path=[SubjFolderlist '/MNINonLinear/fsaverage_LR32k'];
+        native_path=[SubjFolderlist '/MNINonLinear/Native'];
+        vn_file_path=[SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_Atlas_MSMAll_hp' hp '_clean_vn.dscalar.nii'];
+        vn_smooth_file_path=[SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_Atlas_MSMAll_hp' hp '_clean_vn_' num2str(FinalSpatialSmoothingFWHM) '.dscalar.nii'];
+
+        left_msmsulc_msmall_cmd=['wb_command -surface-sphere-project-unproject ' fs_path '/' subj '.L.sphere.32k_fs_LR.surf.gii ' native_path '/' subj '.L.sphere.MSMSulc.native.surf.gii ' native_path '/' subj '.L.sphere.MSMAll.native.surf.gii ' fs_path '/' subj '.L.sphere.MSMSulc_MSMAll.32k_fs_LR.surf.gii'];
```

Comment:
If all you are doing is running the command, don't assign it to a variable.  Also, please use `system`, not `unix`, workbench is cross-platform, and using `unix` to run commands from a language that is cross-platform looks dumb.